ui/src/submissions/common/components/SuggesterField.jsx

```diff
@@ -12,6 +16,10 @@ class SuggesterField extends Component {
     this.recordFieldPopulated = false;
   }
 
+  static getSuggestionControlNumber(suggestion) {
```

Comment:
is this on purpose?  https://github.com/inspirehep/inspirehep/pull/1381/files#diff-3d8676fe3ea027dff622d0b717aaa7ccR6

Reply:
no leftover.